src/assets/fonts/source-sans-pro/generator_config.txt

```diff
@@ -0,0 +1,5 @@
+# Font Squirrel Font-face Generator Configuration File
```

Comment:
Do we actually need this file :thinking: ?

Reply:
Do you want to remove those files? Actually in titillium this info was helpfully, i didn't spent time for figuring which service/approach was used before. Also I guess this file gives possibility to continue edit fonts setting but not create from scratch again